cmd/harmony/main.go

```
@@ -226,10 +223,7 @@ func main() {
 		}
 		currentNode.NodeConfig.SetShardGroupID(p2p.GroupIDBeacon)
 	} else {
-		var beacondb ethdb.Database
-		if *dbSupported {
-			beacondb, _ = InitLDBDatabase(*ip, *port, *freshDB, true)
-		}
+		beacondb, _ := InitLDBDatabase(*ip, *port, *freshDB, true)
```

Let's not ignore errors (2).

---

done